diff --git a/dist/lib/dprint/dprint.js b/dist/lib/dprint/dprint.js
index 1574a7f0b7010f2cbe23123c6524fccb0bb23b35..c2dcf1d350654b5ee1ccb1237ede0c9c10067519 100644
--- a/dist/lib/dprint/dprint.js
+++ b/dist/lib/dprint/dprint.js
@@ -117,7 +117,8 @@ function getFormatter(filePath, configName, configFile, log = true) {
             return formatter;
         }
         else if (isPluginName(configName) && log) {
-            console.warn("File %s not supported by %s", filePath, pluginsName[configName]);
+            // silenced to reduce noise
+            // console.warn("File %s not supported by %s", filePath, pluginsName[configName]);
         }
     }
     else if (isPluginName(configName) && log) {
